enhancements/network/persistent-ips-sdn-secondary-networks.md

```diff
@@ -807,3 +807,99 @@ allocations, it is probable the time it takes to provision a pod to grow.
 
 Implementing a new CNI IPAM plugin - or refactoring whereabouts to address its
 shortcomings - would be a major effort.
+
+### Separate IPAMLeases controller
+
+This alternative approach requires a separate controller to monitor both
+`PersistentIP` and `NetworkAttachmentDefinition` objects.
+
+KubeVirt would express its intent in having a `PersistentIP` for its VM
+interface (1 `PersistentIP` for each interface) by creating a CR; the new
+controller would learn about KubeVirt's desire, calculate an IP for it, and
+update the `PersistentIP` status with this information.
+
+KubeVirt VM templating engine would block the VM creating until it sees the
+persistent IP allocation available in the CR; once the IP is available in the
+CR, KubeVirt would be able to template the pod accordingly, requesting that IP
+in the interface's `NetworkSelectionElements`.
+
+This flow is portrayed in the following sequence diagram:
```

Comment:
Diagrams talk about  `IPAMLease` while description talk about `PersistentIP` we need just one of them.

Reply:
We're speaking about `PersistentIP` throughout the entire presentation. 

But you're right I should probably align the names everywhere.